bootstrap

```diff
@@ -15,7 +15,7 @@ if [ -f /etc/debian_version ]; then
     fi
 fi
 if [ -f /etc/redhat-release ]; then
-    for package in python3-pip python3-virtualenv python3-devel libevent-devel libxml2-devel libxslt-devel zlib-devel; do
+    for package in python3-pip python3-venv python3-devel libevent-devel libxml2-devel libxslt-devel zlib-devel; do
```

Comment:
testing OK on Ubuntu - the fix works
testing on RH family distros, the package name has not changed (it is still `python3-virtualenv`)
(checked fedora, RHEL8 & 9, centos-stream)

```suggestion
    for package in python3-pip python3-virtualenv python3-devel libevent-devel libxml2-devel libxslt-devel zlib-devel; do
```

Reply:
ok, thanks